src/Microsoft.WSMan.Management/CredSSP.cs

```diff
@@ -440,7 +440,7 @@ public class EnableWSManCredSSPCommand : WSManCredSSPCommandBase, IDisposable/*,
 
         //private const string DelegateComputerParam = "DelegateComputer";
         //private String[] delegatecomputer;
-        //private RuntimeDefinedParameterDictionary dyanmicParameters = new RuntimeDefinedParameterDictionary();
+        //private RuntimeDefinedParameterDictionary dynamicParameters = new RuntimeDefinedParameterDictionary();
```

Comment:
This code is commented out. I verified that all usages were replaced. @lzybkr do we want to remove this unused code?